.circleci/config.yml

```diff
@@ -70,9 +70,32 @@ jobs:
             sudo pip install --upgrade setuptools
             sudo pip install awsebcli --upgrade
       - run:
-          name: Deploying
-          command: eb deploy JadeServiceRunner-env
+          name: Ensure Stack
+          command: cd cfn && init-staging.sh
```

Comment:
does this directory reset after this `run`?

might need to `cd -` after? not sure how it works in circleci